scripts/core/subAllocationsProperties.yml

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Extension for querying sub-allocations properties."
+version: "1.5"
+--- #--------------------------------------------------------------------------
+type: macro
+desc: "Sub-Allocations Properties Extension Name"
+version: "1.5"
+name: $X_SUB_ALLOCATIONS_EXP_NAME
+value: '"$X_experimental_sub_allocations"'
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Sub-Allocations Properties Extension Version(s)"
+version: "1.5"
+name: $x_sub_allocations_exp_version_t
+etors:
+    - name: "1_0"
+      value: "$X_MAKE_VERSION( 1, 0 )"
+      desc: "version 1.0"
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Properties returned for a sub-allocation"
+version: "1.5"
+name: $x_sub_allocations_exp_properties_t
+base: $x_base_properties_t
+members:
+    - type: "const void*"
+      name: ptr
+      desc: "[out][optional] base address of the sub-allocation"
+    - type: size_t
+      name: size
+      desc: "[out][optional] size of the allocation"
+details:
+    - "This structure may be passed in $x_sub_allocations_properties_exp_desc_t, when querying for the properties of the sub-allocations of an allocation."
```

Comment:
Can you check that this detail is saying what you'd like it to say?

* The spelling of `$x_sub_allocations_properties_exp_desc_t` doesn't look quite right - should this be `$x_sub_allocations_exp_desc_t` instead?
* Is this structure "passed in" or "returned from" the query?
* If the structure is purely returned from the query does it need to be derived from the base properties and hence have an stype and pNext or can this be omitted?
    * Note that there is an stype and a pNext for the sub-allocations descriptor `$x_sub_allocations_exp_desc_t` if we need to extend the chain to query other properties.